README.md

```diff
@@ -19,11 +19,9 @@ Tools V2
 
 **1. Creating a tracelogging graph:**
 
-- Make a directory "tracelogging"
-- Download the content of the "website" folder and put it in the "tracelogging" folder
-- Copy the website/server.py file to the directory containing the trace files (/tmp on linux, the working directory in windows)
-- Run "python server.py" from that directory.
-- Navigate with a browser to "file:///PATH_TO/tracelogger.html"
+- Checkout the tracelogger repo, or download just the content of the "website" folder
+- Run "python <path-to-website-dir>/server.py"
+- Navigate with a browser to "http://localhost:8000/tracelogger.html"
```

Comment:
I was planning on doing that, but realized it wasn't a completely simple change.

Sure, we could rename tracelogger.html to index.html, but you'd still have to type that into the url. What you really want is for http://localhost:8000/ to redirect to http://localhost:8000/tracelogger.html (or index.html, it doesn't really matter). But that URL is already used internally to retrieve the list of tl-* files. So you'd need to use something else for that purpose instead. (Fancy way: send some sort of 'Accept: application/json' header and when that is detected, send back a JSON file listing. Much easier way: use /tracelogger.html?list-data-files=1 or /data-files).

At any rate, it would require policy opinions that I'd rather leave to you. :-)